idempotence4j-mariadb/src/main/resources/db/idempotence4j/mariadb/V20220429110000__add_idempotent_action_type_client_index.sql

```diff
@@ -0,0 +1,2 @@
+CREATE INDEX IF NOT EXISTS IDX_IDEMPOTENT_ACTION_TYPE_CLIENT
```

Comment:
I was just googling to see if there's a mariadb equivalent of concurrently - looks like adding `ALGORITHM=INPLACE LOCK=NONE` may do the trick?